File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int N = scan.nextInt();

        int A[]= new int[N];
        for(int i=0; i<N; i++)
            A[i] = scan.nextInt();

        System.out.println(Arrays.stream(A).min().getAsInt() + " " + Arrays.stream(A).max().getAsInt());
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main()
{
    int i, N, a, max, min;

    scanf("%d\n", &N);
    scanf("%d", &a);
    max = a;
    min = a;
    for (i = 0; i < N - 1; i++)
    {
        scanf("%d", &a);
        if (a > max)
            max = a;
        else if (a < min)
            min = a;
    }
    printf("%d %d", min, max);

    return 0;
}